src/components/ArticlePage.jsx

```diff
@@ -10,16 +12,34 @@ import Comments from './Comments';
 function ArticlePage() {
     const {article_id} = useParams();
     const [article, setArticle] = useState();
+    const [voteCount, setVoteCount] = useState(0);
     const [loading, setLoading] = useState(true);
+    const [vote, setVote] = useState(false);
+    const [downVote, setDownVote] = useState(false);
 
     useEffect(() => {
         fetchArticleById(article_id)
         .then((articleResponse) => {
             setArticle(articleResponse);
+            setVoteCount(articleResponse.votes);
             setLoading(false)
         })
     }, [])
 
+    function incrementVote(){
+        setVote(true);
+        setDownVote(false);
+        setVoteCount(1);
+        patchVotes(article_id, 1);
+    }
+
```

Comment:
You should also make sure that you are handling the vote change with optimistic rendering. To recap optimistic rendering should:
- updated the vote state before you make the request to your backend api
- if the request to your api is unsuccessful and has an error you will need to revert the vote amount back to it's original value.

You can see more about optimistic rendering in the notes here - https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering#optimistic-rendering